src/test/java/org/opentripplanner/middleware/testutils/CommonTestUtils.java

```diff
@@ -5,12 +5,6 @@
 import java.io.IOException;
 
 public class CommonTestUtils {
-    /**
-     * Whether the end-to-end environment variable is enabled.
-     */
-    public static final boolean IS_END_TO_END = getBooleanEnvVar("RUN_E2E");
-
-    private static final String TEST_RESOURCE_PATH = "src/test/resources/org/opentripplanner/middleware/";
```

Comment:
I put it in here because it seemed like it was related to items related the overall test environment. The CommonTestUtils seem more like methods that help obtain things, but the variable in question could be considered something that is more part of the overall test environment.